augur/data/schema-export-root-sequence.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://nextstrain.org/schemas/dataset/root-sequence",
+  "title": "Nextstrain root-sequence sidecar for datasets",
+  "description": "Typically produced by Augur and consumed by Auspice.  Applicable to the `--root-sequence` output of `augur export v2` as well as the `--output-sequence` option of `augur export v1`.",
+  "oneOf": [
+    {
+      "description": "An empty object",
+      "type": "object",
+      "properties": {},
+      "additionalProperties": false
+    },
```

Comment:
(maybe outside the scope of this PR, but I think generating an empty JSON file is a bit weird)

Reply:
Agreed it's weird! I added a comment.

I was trying to stick to being descriptive of existing forms in this PR but would definitely support removing it as a possibility in another PR. I think it would even be ok to decide that the empty object produced by `augur export v1` is invalid.